src/js/modules/index.js

```diff
@@ -0,0 +1,58 @@
+import { displayText } from '../utils/index.js';
+import { INIT_DISPLAY_TEXT, OPERATIONS } from '../constants/index.js';
+import { validateDigit, validateOperation } from '../validations/index.js';
+
+export const handleDigits = (e) => {
+  const clickedNumber = e.target.innerText;
+
+  if (validateDigit()) return;
+  
+  if (displayText.innerText === INIT_DISPLAY_TEXT) {
+    displayText.innerText = clickedNumber;
+    return
+  };
+
+  displayText.innerText += clickedNumber;
+}
+
+export const handleOperations = (e) => {
+  const clickedOperation = e.target.innerText;
+  const operations = displayText.innerText.split('').filter((text) => isNaN(text));
```

Comment:
```suggestion
  const operations = displayText.innerText.split('').filter(isNaN);
```

isNaN을 래핑하지 않아도 동일하게 동작해요!

Reply:
@jho2301 
앗 그렇군요.. 이 부분도 반영했습니다 :)